### PR TITLE
Headers

### PR DESCRIPTION
**Pull request by XhmikosR**
* Move referrer header in Hapi's security object
* Switch to blankie for CSP

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,9 +25,20 @@ const hbsPartialFile = handlebarsPartialFile({
 
 hbsPartialFile.registerDirectory('badge', 'svg');
 
-// eslint-disable-next-line quotes
-const CSP_HEADER = "default-src 'none'; base-uri 'none'; form-action 'self'; frame-src 'self'; frame-ancestors 'self'; img-src 'self'; manifest-src 'self'; style-src 'self'";
-const REFERRER_HEADER = 'no-referrer, strict-origin-when-cross-origin';
+const CSP = {
```
Comment (mozfreddyb) [inline index.js]: I wonder how blankie deals with object property ordering, given this here is alphabetically sorted (and that they don't have an order in JavaScript in the first place).

I think it's somewhat important the default-src is sent first. Does that happen with this patch? :thinking:

File: index.js [hunk 2]
```diff
@@ -38,12 +51,16 @@ const REFERRER_HEADER = 'no-referrer, strict-origin-when-cross-origin';
             maxAge: 31536000,
             preload: true
           },
+          referrer: 'strict-origin-when-cross-origin',
```
Comment (mozfreddyb) [inline index.js]: Before it was "no-referrer, strict-origin-when-origin", why the change?

File: index.js (hunk 1 above)
Comment (XhmikosR) [reply index.js]: Are you 100% sure the order matters? This patch doesn't send `default-src` first, but I could change the order. As long as it does matter

File: index.js (hunk 2 above)
Comment (XhmikosR) [reply index.js]: I couldn't find the exact thing we were using in the Hapi security options, so I went with the closest one... https://hapi.dev/api/?v=18.4.0#-routeoptionssecurity

Comment (XhmikosR) [reply index.js]: So, after reading the docs, it seems that `no-referrer` is a fallback for when browsers don't support  `strict-origin-when-origin`. Unfortunately, it doesn't seem hapi allows us to do this:

```
[1] "referrer" must be a boolean
[2] "referrer" must be one of [no-referrer, no-referrer-when-downgrade, unsafe-url, same-origin, origin, strict-origin, origin-when-cross-origin, strict-origin-when-cross-origin]
```

We could try asking upstream for this, not sure if they'll accept this.

File: index.js (hunk 1 above)
Comment (mozfreddyb) [reply index.js]: The order doesn't matter for browser parsing, actually. Just for reading it :-) Nevermind..

File: index.js (hunk 2 above)
Comment (mozfreddyb) [reply index.js]: I couldn't find any modern browser that doesn't support strict-origin-when-cross-origin.

I believe the policy was written when the spec wasn't completely implemented by Chrome.

Comment (XhmikosR) [reply index.js]: Then if you think it's OK I guess we can move with merging this PR?

**Comment by XhmikosR** [comment]
@mozfreddyb I'd like to get this merged if you agree with the changes.
